tests/revoke_module/test_approve_action.py

```diff
@@ -0,0 +1,17 @@
+from brownie import reverts
+
+
+def test_approve(module, safe, deployer, token, rug_puller, module_set_up):
+    assert token.allowance(safe, rug_puller) > 0
+    tx = module.approve(safe, token, rug_puller, {"from": deployer})
```

Comment:
calling this function `approve` is actually confusing; proposing `revoke`

Reply:
sorted in fa03684d6593df53c8a39a0b0b5c2ccb7b4422fd